domain/domain.go

```diff
@@ -550,11 +551,11 @@ func (c *ddlCallback) OnChanged(err error) error {
 	if err != nil {
 		return err
 	}
-	log.Infof("[ddl] on DDL change, must reload")
+	logutil.Logger(context.Background()).Info("doing  DDL change, must reload")
```

Comment:
s/doing  DDL/doing DDL
BTW, You prefer 'doing' to 'on'?

Reply:
```suggestion
	logutil.Logger(context.Background()).Info("on DDL change, must reload")
```

Reply:
I prefer "performing DDL change" or "ddl change in progress".  on/doing DDL is not a natural way to word this.